app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,28 @@
 class PostsController < ApplicationController
+  def new
+    post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: } }

```

Comment:
- [OPTIONAL] I got a syntax error when I tried to run the app. I think you need to pass the `post` variable defined in line 3 into line  5. Please screenshot of the error.

Screenshot |
:--------------:|
![a1](https://user-images.githubusercontent.com/39523312/167020450-ce81177e-44df-4328-87dd-7f6b078a485a.png)